Makefile

```diff
@@ -155,7 +155,6 @@ docker-build-only-$(1):
 		--build-arg BUILDGOVERSION=$(BUILDGOVERSION) \
 		--build-arg BUILDSTAMP=$(NOW_ISO8601) \
 		-t $$(CONTAINER_NAME):$(CONTAINER_TAG) \
-		-t $$(CONTAINER_NAME):$(CONTAINER_VERSION) \
```

Comment:
I dont want to push two images per release to docker.io, and it's fairly impossible to put an `if` in here. So I just move this second target to the internal release's `CONTAINER_BUILD_EXTRA_ARGS`